docs/sources/panels/value-mappings.md

```diff
@@ -3,14 +3,136 @@ title = "Value mappings"
 weight = 600
 +++
 
-## Value mappings
+# Value mappings
 
-Value mappings come in different types.
+Value mapping concept allow you to replace values or ranges in your visualizations with words or emojis.
 
-* **Range** maps numerical ranges to a display text and color. 
-* **Value** maps text values to a color or different display text.
-* **Special** maps special values like `Null`, `NaN` and boolean values like `true` and `false` to a display text and color.
+Values mapped via value mappings will skip the unit formatting. This means that a text value mapped to a numerical value will not be formatted using the configured unit.
 
-The display text and color are both optional. If you only want to assign colors to text values you can leave the display text empty and the original value will be used for display.
+![Value mappings example](/static/img/docs/value-mappings/value-mappings-example-8-0.png)
 
-Values mapped via value mappings will skip the unit formatting. This means that a text value mapped to a numerical value will not be formatted using the configured unit.
\ No newline at end of file
+If value mappings are present in a panel, then Grafana displays a summary in the side pane of the panel editor.
+
+## Value mapping examples
+
+Value mappings are displayed differently in different visualizations.
+
+### Time series example
+
+Here's an example showing a Time series visualization with value mappings.
+
+![Value mappings time series example](/static/img/docs/value-mappings/value-mappings-summary-example-8-0.png)
+
```

Comment:
This time series example can be removed as the value mappings are not used in the time series panel. Here only the select color scheme (classic palette) is used to define series colors, and the series color is used for the line/area/points.  The value mappings only apply for visualizations that render individual values (with different colors).

Reply:
Except that the value mappings appeared on the axis. Is that not intentional?

Reply:
Yes, for axis values. But no color. So if you want to show case axis values that is then don't set color as that is not used at all. See example in what's new in v7.4 dashboard on play, it has a value mapping example with a graph that show cases value mapping for y axis

Reply:
Okay. What I was trying to do with the examples was keep the same settings and just change the visualization, showing what the same settings would do for each.

Reply:
BTW, sometimes the value mappings disappear when you switch visualizations. I haven't found a pattern to it yet, just suddenly have to redo them all. Super annoying.

Reply:
could be that some visualization do not support it (old graph, heatmap, text panel etc)

Reply:
This one I would use for value time series
![Screenshot from 2021-06-02 19-35-03](https://user-images.githubusercontent.com/10999/120526615-ad6f3c80-c3d9-11eb-8d29-9002b6369d81.png)

Reply:
Pretty! I will add that in, thanks!

Reply:
Added in https://github.com/grafana/website/pull/4630